HISTORY.rst

```diff
@@ -10,6 +10,10 @@ API Changes (Backward-Compatible)
 Bugfixes
 ~~~~~~~~
 
+- CONTINUATION frames sent on closed streams previously caused stream errors
+  of type STREAM_CLOSED. RFC 7540 requires that these be connection errors of
```

Comment:
Am I right in thinking this is RFC 7540 § 6.10? Took me a minute or two to find – worth making the reference more specific?